saleor/account/emails.py

```diff
@@ -33,6 +38,53 @@ def _send_password_reset_email(recipient_email, reset_url, user_id):
     account_events.customer_password_reset_link_sent_event(user_id=user_id)
 
 
+def send_user_change_email_url(redirect_url, user, new_email, token):
+    """Trigger sending a email change email for the given user."""
+    event_parameters = {"old_email": user.email, "new_email": new_email}
+    _send_email_change_email_with_url.delay(
+        new_email, redirect_url, user.pk, token, event_parameters
+    )
+
+
+@app.task
+def _send_email_change_email_with_url(
```

Comment:
I know it sounds strange but it would be better to use "request email change" here, so the name would be `_send_request_email_change_email`. No need to use `with_url` as this is legacy naming I guess.